bindings/wasm/src/account/update/create_method.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use js_sys::Promise;
+use std::rc::Rc;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+use identity::account::Update;
+use identity::account::UpdateError::MissingRequiredField;
+use identity::account::{Account, MethodSecret};
+use identity::did::MethodScope;
+use identity::did::MethodType;
+use wasm_bindgen::__rt::WasmRefCell;
+
+use crate::account::wasm_account::WasmAccount;
+use crate::account::wasm_method_secret::WasmMethodSecret;
+use crate::did::WasmMethodScope;
+use crate::did::WasmMethodType;
+
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  /// Adds a new verification method to the DID document.
+  #[wasm_bindgen(js_name = createMethod)]
+  pub fn create_method(&mut self, options: &CreateMethodOptions) -> Result<Promise> {
+    let account: Rc<WasmRefCell<Account>> = Rc::clone(&self.0);
+
+    let method_type: MethodType = options
+      .methodType()
+      .map(|m| m.0)
+      .unwrap_or(MethodType::Ed25519VerificationKey2018);
```

Comment:
@abdulmth raised the point that we don't get compilation errors in the bindings with builders if the Rust `Account` adds new fields to them, which we would when using the update structs directly.

I'm still uncertain if we should make them public for Rust too but I'm starting to lean that direction, mostly held back by the status quo of leaving them un-exported to reduce the API surface and ensure there is only one "canonical" method of updating `Accounts` to avoid confusion.